src/core/Object3D.js

```diff
@@ -303,10 +303,10 @@ Object3D.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 
 	lookAt: function () {
 
-		// This method does not support objects with rotated and/or translated parent(s)
-
-		var m1 = new Matrix4();
-		var vector = new Vector3();
+		const m1 = new THREE.Matrix4();
```

Comment:
Please avoid the usage of the `THREE` namespace in the source. Besides, use `var` and not `const` or `let`.

Reply:
I'm a little curious why no `const`. THREE is using ES6 modules and the build already translates `const` into `var`. Is this just a project preference? Is it an outdated preference? There's no perf difference and it's considered best practice by nearly every major tech company's style guide. It also prevents lots of errors and scoping issues. Is there a link to some rational for the policy?

Reply:
> Is this just a project preference?

Yes. At least for now. 

/cc @mrdoob